ride-share.rb

```diff
@@ -0,0 +1,57 @@
+drivers = [
+  { 
+    id: "DR0001", 
+    date: [ "3rd Feb 2016", "3rd Feb 2016", "5th Feb 2016" ],
+    cost: [ 10, 30, 45 ],
```

Comment:
This data is hyper optimized for solving the problems that we hand you, but it isn't great for showing people who look at it which data is grouped together. That's fine, I just want you to be aware of how these trade-offs get made.